composer.json

```diff
@@ -78,7 +82,13 @@
     "post-create-project-cmd": [
       "@cleanup",
       "WordPressProject\\composer\\ScriptHandler::createRequiredFiles"
+    ],
+    "code-sniff": [
+      "./vendor/bin/phpcs --config-set installed_paths ./vendor/wp-coding-standards/wpcs",
+      "./vendor/bin/phpcs ./web/wp-content"
+    ],
+    "unit-test": [
+      "./vendor/bin/phpunit tests/unit/*"
```

Comment:
What are your thoughts on using this and dropping the .gitkeep files?

`if [ -d ./web/wp-content ]; then ./vendor/bin/phpcs ./web/wp-content fi`